public/pages/DetectorsList/Components/ListControls/ListControls.tsx

```diff
@@ -39,7 +39,7 @@ interface ListControlsProps {
   onPageClick: (pageNumber: number) => void;
 }
 export const ListControls = (props: ListControlsProps) => (
-  <EuiFlexGroup>
+  <EuiFlexGroup gutterSize="s">
```

Comment:
will this break snapshot test?

Reply:
I've noticed the current development branch has a lot of broken test snapshots. I didn't want to put out multiple PRs that are all updating a lot of the same test snapshots. I could also just put out a PR to update the snapshots, get that merged first, then update any snapshots specific to these changes.

Reply:
Sure, sounds good. So we will use github action to update snapshot when PR published or when code merged?

Reply:
Oh, seem I put wrong comments, should be for https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/42/files

Reply:
The action won't update any snapshots; it just runs all of the unit tests when anything is pushed to development branch (when PR is merged). Used as a final check after the PR is approved & merged.

Reply:
Cool. Is it possible we run test when PR published? So we can know any failure earlier and fix.

Reply:
Moving conversation to the other PR ->

Reply:
I will put out a PR that simply updates test snapshots, though.